### PR TITLE
Solrize member ids in objects and collections

### PR DESCRIPTION
**Pull request by awead**
Solution to #61.

### CODE REVIEW

File: lib/hydra/pcdm/models/concerns/collection_behavior.rb [hunk 1]
```diff
@@ -86,6 +86,13 @@ def ancestor? collection
       false
     end
 
+    def to_solr(solr_doc = Hash.new, opts={})
+      solr_doc = indexing_service.generate_solr_document
+      solr_doc["objects_ssim"] = self.objects.map { |o| o.id }
```
Comment (jcoyne) [inline lib/hydra/pcdm/models/concerns/collection_behavior.rb]: These should go into an IndexingService, not override to_solr

Comment (awead) [inline lib/hydra/pcdm/models/concerns/collection_behavior.rb]: @jcoyne can you point to an example of how that's done?

Comment (jcoyne) [inline lib/hydra/pcdm/models/concerns/collection_behavior.rb]: Check out this commit: https://github.com/curationexperts/alexandria-v2/commit/91f3c61346f99b10b8b374599e1b34c56de2f81b Extend ActiveFedora::IndexingService and override generate_solr_document in it. Then override "Collection.indexer" to return the class you've defined.

**Comment by jcoyne** [comment]
I just want to point out that this could be very slow if there are lots of members.  We're sure we want to accept that penalty, right?  (I think it's okay.)

**Comment by jcoyne** [comment]
An alternative approach would be to get `member_ids` (doesn't instantiate the objects) and classify them (Object vs. Collection) by querying solr for their types. I think the current approach is okay for now, but there's definitely the opportunity to improve if this is slow.

**Comment by awead** [comment]
@jcoyne I agree there's a potential performance issue. We agreed to earmark those but not worry about them during the sprint. I've noted the issue here #83.

**Comment by jcoyne** [comment]
@awead Objects don't have collections, so it should have an indexer that doesn't try to index collections

**Comment by awead** [comment]
@jcoyne I have `unless Hydra::PCDM.object?(object)` to prevent that.

**Pull request merged**